staging/src/k8s.io/apimachinery/pkg/conversion/converter.go

```diff
@@ -40,8 +40,14 @@ type NameFunc func(t reflect.Type) string
 
 var DefaultNameFunc = func(t reflect.Type) string { return t.Name() }
 
+// GenericConversionFunction copies data from a into b. It should return an error if the
+// data cannot be moved. It returns false if the provided objects are not recognized.
 type GenericConversionFunc func(a, b interface{}, scope Scope) (bool, error)
 
+// ConversionFunction moves data from a into b. It should return an error if the data cannot
+// be moved.
+type ConversionFunc func(a, b interface{}, scope Scope) error
```

Comment:
define "move". I know what you mean and it is actually a good term (= `b` might share data structures with `a`, but `a` is not mutated).